packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/impact-analysis/useImpactAnalysis.ts

```diff
@@ -47,43 +44,9 @@ const getImpactAnalysisReportQuery = gql`
   }
 `;
 
-export const useInvokeImpactAnalysis = (
-  editingContextId: string | null,
-  representationId: string | null,
-  toolId: string | null,
-  diagramElementId: string | null,
-  variables: GQLToolVariable[] | null
-): UseInvokeImpactAnalysisValue => {
-  const { loading, data, error } = useQuery<GQLGetImpactAnalysisReportData, GQLInvokeImpactAnalysisToolVariables>(
-    getImpactAnalysisReportQuery,
-    {
-      skip:
-        editingContextId === null ||
-        representationId === null ||
-        toolId === null ||
-        diagramElementId === null ||
-        variables === null,
-      variables: {
-        editingContextId,
-        representationId,
-        toolId,
-        diagramElementId,
-        variables,
-      },
-    }
+export const useInvokeImpactAnalysis = (): UseInvokeImpactAnalysisValue => {
```

Comment:
We probably don't need a hook for that

Reply:
You should add error management and everything as usual here, you would then need this hook as usual :)